packages/security/src/deviceTrust/checks/DebuggerCheck.ts

```diff
@@ -0,0 +1,22 @@
+import { SecurityCheck } from "../SecurityCheck";
+import { SecurityCheckResult } from "../SecurityCheckResult";
+
+declare let cordova: any;
+
+export class DebuggerCheck implements SecurityCheck {
+    public readonly name = "Is Debugger Check";
```

Comment:
static? And for constants names please use UPPER_CASE.

Reply:
-1. That's Java land. Cannot find any example for this in ts[1]. Let's not interfere with style until we have strict style guide. In RainCatcher we were using[2]

[1] https://basarat.gitbooks.io/typescript/docs/const.html
[2] https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines